server/helpers/loadConfig.js

```diff
@@ -69,7 +69,8 @@ const {
   PORT,
   REDIS_URL,
   ADALITE_ENABLE_DEBUGGING,
-  ADALITE_SERVER_URL,
+  HEROKU_APP_NAME,
+  HEROKU_IS_REVIEW_APP,
```

Comment:
just a sanity check - these are injected by heroku? If they're required in local .env, add them to .env.example. If not, ignore this

Reply:
`HEROKU_APP_NAME` is injected by Heroku and `HEROKU_IS_REVIEW_APP` is set by us. I did not add `HEROKU_IS_REVIEW_APP` into `.env.example` as for any local development we just should not set it. As written in the comment in code, `HEROKU_IS_REVIEW_APP` is introduced just because `HEROKU_APP_NAME` is not considered stable, I want to avoid the case when it would suddenly appear for all heroku apps (e.g. staging :) )